lib/index.js

```diff
@@ -243,13 +241,16 @@ export function configureUnifiedLanguageServer(
     const start = Position.create(0, 0)
     const end = document.positionAt(text.length)
 
+    // V8 coverage bug on Dubnium (Node 12).
```

Comment:
Let’s use [unicorn todo comments](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/expiring-todo-comments.md)

When CI is upgraded to include Node 18, I think we drop support for Node 12, right?

```suggestion
    // XXX [engine:node@>16] V8 coverage bug on Dubnium (Node 12).
```